### DISCUSSION TITLE
AttributeError: module 'PIL.Image' has no attribute 'Resampling' when loading a PDF with UnstructuredPDFLoader

### DISCUSSION THREAD

**Discussion by Bhagat028** [Unstructured-IO/unstructured#423] [Q&A]
Hello everyone,

I've encountered an issue while using the UnstructuredPDFLoader and partition functions from the langchain.document_loaders and unstructured.partition.auto modules, respectively. I have successfully installed the following packages:

pip install "unstructured[local-inference]"
pip install unstructured
pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@v0.6#egg=detectron2"
However, I keep getting the following error: AttributeError: module 'PIL.Image' has no attribute 'Resampling'. I've tried installing poppler-utils and other dependencies mentioned in the quick start guide, but the error persists.

I'm seeking assistance from the community in order to resolve this issue. Any help or guidance would be greatly appreciated.

Thank you!

**Comment by MthwRobinson** [comment]
@Eduin2024 - Uninstalling and reinstalling `pillow` or (if you're working on Colab) restarting your runtime may help with this issue.
